packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/createAccount.js

```diff
@@ -120,12 +151,22 @@ const CreateAccount = props => {
                 <div className={classes.subscribe}>
                     <Checkbox
                         field="subscribe"
-                        label="Subscribe to news and updates"
+                        label={formatMessage({
+                            id: 'checkoutPage.subscribe',
+                            defaultMessage: 'Subscribe to news and updates'
+                        })}
                     />
                 </div>
                 <div className={classes.actions}>
-                    <Button disabled={isDisabled} type="submit">
-                        {'Create Account'}
+                    <Button
+                        disabled={isDisabled}
+                        type="submit"
+                        className={classes.create_account_button}
```

Comment:
Will probably want to add `composes ... from Button` to the custom css class. Forgot to tell you! My bad.